config_template.cfg

```diff
@@ -1346,7 +1346,7 @@ CFL_REDUCTION_ADJTURB= 0.01
 
 % -------------------- NEMO NUMERICAL METHOD DEFINITION -----------------------%
 %
-% Mixture transport properties (WILKE,GUPTA-YOS,CHAPMANN-ENSKOG)
+% Mixture transport properties (WILKE,GUPTA-YOS,CHAPMANN-ENSKOG, SUTHERLAND)
```

Comment:
Overall looks good! I think this will be a good addition for running the lower temperature experimental verification cases with NEMO, where the T_{\inf} is low but he Ma_{\inf} is high.